.github/workflows/pr-dev-to-main.yml

```diff
@@ -0,0 +1,33 @@
+name: Create Pull Request from dev to main
+
+on:
+  workflow_dispatch:
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Fetch All
+        run: git fetch --all
+
+      - name: Get Latest Tag
+        id: previoustag
+        uses: WyriHaximus/github-action-get-previous-tag@v1.4.0
+        with:
+          fallback: v1.0.0
+
+      - name: Create Dev to Main Pull Request
+        run: |
+          git checkout dev
+          git pull
+          git checkout -b dev-${{ steps.previoustag.outputs.tag }}
```

Comment:
use `git switch -c`. Git switch is a replacement of checkout

Reply:
Alright